src/solver/optimisation/adequacy_patch_csr/adq_patch_post_process_list.cpp

```diff
@@ -21,9 +21,11 @@ AdqPatchPostProcessList::AdqPatchPostProcessList(const AdqPatchParams& adqPatchP
       problemeHebdo_, areas, sheddingPolicy, splxOptimization, thread_number));
     post_process_list.push_back(std::make_unique<CurtailmentSharingPostProcessCmd>(
       adqPatchParams, problemeHebdo_, areas, thread_number_));
-    // Here a post process particular to adq patch
-    post_process_list.push_back(std::make_unique<DTGmarginForAdqPatchPostProcessCmd>(
-      adqPatchParams, problemeHebdo_, areas, thread_number));
+    post_process_list.push_back(std::make_unique<UpdateMrgPriceAfterCSRcmd>(problemeHebdo_,
+                                                                            areas,
+                                                                            thread_number));
+    post_process_list.push_back(std::make_unique<DTGnettingAfterCSRcmd>(
+      problemeHebdo_, areas, thread_number));
```

Comment:
Here we separate 2 post processes : 
- the update of **MRG price** depending on the value of **ENS**, after the step of CSR. 
  Indeed, for certain hours, step of CSR possibly changed the **ENS**, and **MRG price** has to be changed subsequently, depending on the new value of **ENS**.
- Compute and set new variables (for **ELIA**'s need)